ui/src/App.tsx

```diff
@@ -166,10 +162,10 @@ const App = () => {
           <TextField style={{margin:"10px"}} id="outlined-basic" label="Armspan" variant="outlined" />
           <Divider />
           <Typography style={{margin:"10px"}}>2. Upload the route image</Typography>
-          <Button style={{margin:"10px"}} variant="contained" color="secondary" onClick={() => { handleOpenImage()}}> Upload File <input
-    type="file"
-    style={{ display: "none" }}
-  /></Button>
+          <Button style={{margin:"10px"}} variant="contained" color="secondary" 
+          onClick={() => {handleOpenImage()}}> 
+            Upload File <input type="file" style={{ display: "none" }}/>
```

Comment:
If you are starting a new tag, it is good practice for it to be on the left-hand side so it is easy to identify where it is when editing code. Move the <input> tag and info to the next line